columnflow/tasks/plotting.py

```diff
@@ -98,60 +94,92 @@ def run(self):
         ]
         category_inst = self.config_inst.get_category(self.branch_data.category)
         leaf_category_insts = category_inst.get_leaf_categories() or [category_inst]
-        sub_process_insts = {
-            process_inst: [sub for sub, _, _ in process_inst.walk_processes(include_self=True)]
-            for process_inst in process_insts
-        }
 
         # histogram data per process copy
         hists = {}
```

Comment:
I think adding a type hint here might actually help in the future

```suggestion
        hists: dict[od.Config, dict[od.Process, hist]] = {}
```